# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/BasicEvents.h
# Commit: f8a2a169b296
# Full Hash: f8a2a169b2969d29e04733ae27a56459e5359973
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-05-16 09:19:19
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Revert "Bug 1550462 - part 3: Enable `pointerrawupdate` event r=smaug" for causing mochitest failures @ test_pointerrawupdate_event_count.html
#   
#   This reverts commit eb27445e0352d6b4df540adb40e21f72fc052e80.
#   
#   Revert "Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar"
# ==============================================================================

diff -r 8d25773fc82f -r f8a2a169b296 widget/BasicEvents.h
--- a/widget/BasicEvents.h	Fri May 16 06:20:47 2025 +0000
+++ b/widget/BasicEvents.h	Fri May 16 09:30:09 2025 +0300
@@ -454,9 +454,9 @@
         break;
       case ePointerEventClass:
         mFlags.mCancelable =
-            (mMessage != ePointerRawUpdate && mMessage != ePointerEnter &&
-             mMessage != ePointerLeave && mMessage != ePointerCancel &&
-             mMessage != ePointerGotCapture && mMessage != ePointerLostCapture);
+            (mMessage != ePointerEnter && mMessage != ePointerLeave &&
+             mMessage != ePointerCancel && mMessage != ePointerGotCapture &&
+             mMessage != ePointerLostCapture);
         mFlags.mBubbles =
             (mMessage != ePointerEnter && mMessage != ePointerLeave);
         break;
@@ -931,12 +931,12 @@
       case ePointerEventClass:
         // All pointer events are composed
         mFlags.mComposed =
-            mMessage == ePointerRawUpdate || mMessage == ePointerMove ||
             mMessage == ePointerClick || mMessage == ePointerAuxClick ||
             mMessage == eContextMenu || mMessage == ePointerDown ||
-            mMessage == ePointerUp || mMessage == ePointerCancel ||
-            mMessage == ePointerOver || mMessage == ePointerOut ||
-            mMessage == ePointerGotCapture || mMessage == ePointerLostCapture;
+            mMessage == ePointerMove || mMessage == ePointerUp ||
+            mMessage == ePointerCancel || mMessage == ePointerOver ||
+            mMessage == ePointerOut || mMessage == ePointerGotCapture ||
+            mMessage == ePointerLostCapture;
         break;
       case eTouchEventClass:
         // All touch events are composed
@@ -1011,7 +1011,6 @@
         aEventTypeArg.EqualsLiteral("pointerout") ||
         aEventTypeArg.EqualsLiteral("pointerenter") ||
         aEventTypeArg.EqualsLiteral("pointerleave") ||
-        aEventTypeArg.EqualsLiteral("pointerrawupdate") ||
         aEventTypeArg.EqualsLiteral("gotpointercapture") ||
         aEventTypeArg.EqualsLiteral("lostpointercapture") ||
         // touch events